src/tests/mocks/erc2981_mocks.cairo

```diff
@@ -0,0 +1,45 @@
+#[starknet::contract]
+pub(crate) mod ERC2981Mock {
+    use openzeppelin::introspection::src5::SRC5Component;
+    use openzeppelin::token::common::erc2981::ERC2981Component;
+    use starknet::ContractAddress;
+
+    component!(path: ERC2981Component, storage: erc2981, event: ERC2981Event);
+    component!(path: SRC5Component, storage: src5, event: SRC5Event);
+
+    #[storage]
+    struct Storage {
+        #[substorage(v0)]
+        erc2981: ERC2981Component::Storage,
+        #[substorage(v0)]
+        src5: SRC5Component::Storage
+    }
+
+    #[event]
+    #[derive(Drop, starknet::Event)]
+    enum Event {
+        #[flat]
+        ERC2981Event: ERC2981Component::Event,
+        #[flat]
+        SRC5Event: SRC5Component::Event
+    }
+
+
+    #[abi(embed_v0)]
```

Comment:
This should be on top of the Storage struct for consistency with the library.

Reply:
Ok, I have move this part